### ISSUE TITLE
Browser IE11 on Windows 10 seen as generic

### ISSUE THREAD

**Issue by wangkanai** [wangkanai/Detection#62]
### User Agent
```
Mozilla/5.0 (Windows NT 10.0; WOW64; Trident/7.0; rv:11.0) like Gecko
```
![image](https://user-images.githubusercontent.com/10666633/36267325-2c72cb98-12a6-11e8-8810-adaed6f2e50f.png)

**Comment by wangkanai** [comment]
Getting better with 0cfbbce

![image](https://user-images.githubusercontent.com/10666633/72797355-89dba400-3c73-11ea-9d4d-6917bac232cf.png)

**Comment by wangkanai** [comment]
![image](https://user-images.githubusercontent.com/10666633/72816625-0cc42500-3c9b-11ea-84fd-b6ef86c6bcf7.png)
Need the Browser Resolver rewrite to finally close this bug.

**Comment by wangkanai** [comment]
Bug finally fixed with pr #234 

![image](https://user-images.githubusercontent.com/10666633/72902723-8f61e880-3d5e-11ea-8a92-aad566e14128.png)
